src/database/repos/languageRepo.ts

```diff
@@ -24,6 +24,9 @@ export const languageRepo = {
     return database.openSheet(this.sheetTitle, this.columns);
   },
 
+  /**
+   * @returns A list of all languages, including "Other"
```

Comment:
What convention are you following for these comments? Do you use any VSCode extensions around these?

Reply:
JSDoc comments, if you just type `/**` and hit enter in VS Code, it will automatically generate a comment block with a description and `@param` for each argument. You can also type `@` and ctrl+space to get all the annotated keywords.

Reply:
Good to know. Thanks